astronomy-pipeline/astronomy_pipeline.py

```diff
@@ -1,11 +1,15 @@
-"""Full ETL pipeline for the astronomy API data."""
+"""Full ETL pipeline lambda for the astronomy API data."""
+
+import logging
 
 from astronomy_extract import extract_weekly_astronomy_data
 from astronomy_transform import transform_astronomy_data
 from astronomy_load import upload_astronomy_data
 
-if __name__ == "__main__":
 
+def lambda_handler(event=None, context=None) -> None:
+    '''Runs the notification pipeline'''
+    print('Running Lambda')
```

Comment:
Could this print statement be removed in favour of logging now it's going to prod?